hack/update-codegen-dockerized.sh

```diff
@@ -129,12 +129,18 @@ git checkout HEAD -- hack/boilerplate/license_header.raw.txt
 # Download vendored modules to the vendor directory so it's easier to
 # specify the search path of required protobuf files.
 go mod vendor
+# In Go 1.14, vendoring changed (see release notes at
+# https://golang.org/doc/go1.14), and the presence of a go.mod file specifying
```

Comment:
Maybe we could add "the presence of the vendor directory" is also one of the conditions that make go command default to -mod=vendor, thus easier to understand how renaming vendor fixes it.

Reply:
done, thanks